Sources/IosAnalyticsDebugger/Resources/BarDebugger.xib

```diff
@@ -94,8 +95,9 @@
                 <constraint firstItem="hmy-Gf-lbs" firstAttribute="leading" secondItem="vUN-kp-3ea" secondAttribute="leading" id="3k1-zj-n8k"/>
                 <constraint firstItem="8pX-Qi-zk8" firstAttribute="leading" secondItem="vUN-kp-3ea" secondAttribute="leading" id="Oa0-Ti-vCR"/>
                 <constraint firstItem="vUN-kp-3ea" firstAttribute="trailing" secondItem="hmy-Gf-lbs" secondAttribute="trailing" id="OeS-jd-71c"/>
-                <constraint firstItem="vUN-kp-3ea" firstAttribute="bottom" secondItem="hmy-Gf-lbs" secondAttribute="bottom" id="XN1-PQ-Blv"/>
+                <constraint firstAttribute="bottom" secondItem="hmy-Gf-lbs" secondAttribute="bottom" id="XN1-PQ-Blv"/>
                 <constraint firstItem="vUN-kp-3ea" firstAttribute="trailing" secondItem="8pX-Qi-zk8" secondAttribute="trailing" id="lz9-j8-Arz"/>
+                <constraint firstItem="hmy-Gf-lbs" firstAttribute="top" secondItem="iN0-l3-epB" secondAttribute="top" priority="750" id="pgA-tm-Uf9"/>
```

Comment:
Add a top constraint so that the view's height matches the height of the visible bar UI.